### DISCUSSION TITLE
Osu!Lazer turns on only once, after which it does not open.

### DISCUSSION THREAD

**Discussion by MrScoundrel** [ppy/osu#31243] [Q&A]
### Type

Crash to desktop

### Bug description

I can only open the game once, after which I need to restart the computer for it to turn on again. I reinstalled, but the problem didn't go away.

### Version

Lazer 2024.1219.2

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/18218631/compressed-logs.zip)

**Comment by bdach** [comment]
Can you attach a video of what this looks like? Also, can you check for relevant errors in windows event viewer?

Please attach Windows Event Viewer info by performing the following steps:
1. Type `Event viewer` in the Windows search bar (shortcut: <kbd>Windows</kbd>+<kbd>S</kbd>) and press <kbd>Enter</kbd>, or open the *Run...* button (<kbd>Windows</kbd>+<kbd>R</kbd>), type in `eventvwr` and press <kbd>Enter</kbd>.
2. The *Event viewer* window will appear. Expand *Windows Logs* on the left and choose *Application*. It might take a while to load.
3. In the *Actions* pane on the right click *Save All Events As...*, then choose a location to save to.
4. On the *Display Information* popup, select the second option, then the checkbox to show all available languages, then *English (UK)* (US would also do fine).
5. Append the resulting `.evtx` file to this issue.

**Reply by MrScoundrel** [reply]
I'm sorry, but the site does not allow you to send the file of this extension, can you tell me how to do it?

**Reply by bdach** [reply]
Put it in a zip.

**Reply by MrScoundrel** [reply]
[File events.zip](https://github.com/user-attachments/files/18228926/File.events.zip)

**Reply by bdach** [reply]
Error seems to be

```
Description: The process was terminated due to an unhandled exception.
Exception Info: System.UnauthorizedAccessException: Access to the path is denied.
   at System.IO.Pipes.NamedPipeServerStream.Create(String pipeName, PipeDirection direction, Int32 maxNumberOfServerInstances, PipeTransmissionMode transmissionMode, PipeOptions options, Int32 inBufferSize, Int32 outBufferSize, PipeSecurity pipeSecurity, HandleInheritability inheritability, PipeAccessRights additionalAccessRights)
   at System.IO.Pipes.NamedPipeServerStream..ctor(String pipeName, PipeDirection direction)
   at osu.Framework.Platform.NamedPipeIpcProvider.Bind()
   at osu.Framework.Platform.DesktopGameHost.ensureIPCReady()
   at osu.Framework.Platform.DesktopGameHost.get_IsPrimaryInstance()
   at osu.Desktop.Program.Main(String[] args)
```

which indicates a permission problem. Are you running the game on an user account that does not have full privileges or something? Work/school PC, parental controls, anything like that?

**Reply by MrScoundrel** [reply]
No, nothing like that, it's my laptop, and I didn't set up parental controls.
